src/main/java/frc/robot/commands/Drive/LongRunAuto.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Drive;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class LongRunAuto extends SequentialCommandGroup {
+  /** Creates a new LongRunAuto. */
+  public LongRunAuto(DrivetrainSubsystem drive) {
```

Comment:
Is this the "drive over charge station" command?  Should it be named that instead?